### DISCUSSION TITLE
Foundry installation: Forge commands not working

### DISCUSSION THREAD

**Discussion by Kuusho** [Cyfrin/foundry-full-course-cu#1007] [General]
I am trying to install foundry from my command line on Kubuntu (Linux Distro). After installing foundry, I am unable to call:

`forge --version`

the error shown is:

`ZOE ERROR (from forge): zoeParseOptions: unknown option (--version)
ZOE library version 2013-02-16`

It seems the binary being called by `forge` is not the correct one. Has anyone faced this issue and how did you work around it?

**Comment by 7gbice** [comment]
On terminal type >
alias forge=~/. foundry/bin/forge
Thereafter you can type >
forge --version

**Reply by Kuusho** [reply]
I ended up deleting everything in the foundry directory and reinstalling it through my VScode terminal instead of my main terminal.

It worked

**Reply by genkisudo** [reply]
Hey, I getting the same error message, noobs Q: how did you delete everything in foundry directory?

**Reply by Kuusho** [reply]
go to where the foundry folder is and use `rm -r foundry/`
